src/App/PropertyLinks.cpp

```diff
@@ -833,6 +833,18 @@ unsigned int PropertyLinkList::getMemSize(void) const
 }
 
 DocumentObject *PropertyLinkList::find(const std::string &name, int *pindex) const {
+    if(_lValueList.size() <= 10) {

```

Comment:
This could use a comment so that future readers of the code can understand the "magic number" without having to dig into a git blame.